src/components/MainPage/Importers.vue

```diff
@@ -69,12 +71,16 @@ export default {
     ...mapState({
       scrapers: (state) => state.Scrapers.scrapers,
     }),
-    ...mapGetters([
-      'scrapersWithId',
-      'importers',
-    ]),
+    ...mapGetters(['scrapersWithId', 'importers']),
+  },
+  created() {
+    ipcRenderer.send('getScrapers');
+    ipcRenderer.on('getScrapers-reply', (event, scrapers) => {
```

Comment:
I saw in the `AddScraper.vue` you used `ipcRenderer.sendSync` and get the answer to the `const encrypted`.
Why do you use `send` and `on` here? You can't use `sendSync`?

Reply:
I'm guessing it depends on whether the action in the `main` process is `async`.
This is a low-level use of `node` event loop...